pmoves/services/creator-operator/attribution.py

```
@@ -0,0 +1,28 @@
+"""CGP attribution + teaching-summary helpers."""
+
+
+def build_cgp_point(result: dict, workorder: dict, *, model_id: str, license: str) -> dict:
```

Ruff (0.15.15)

[error] Function argument `license` is shadowing a Python builtin

(A002)